legion/engine/llri-vk/detail/adapter.cpp

```diff
@@ -121,7 +121,7 @@ namespace llri
         if (r != VK_SUCCESS)
             return internal::mapVkResult(r);
 
-        capabilities->presentModes = std::vector<present_mode_ext>{};
+        capabilities->presentModes.resize(count);
         for (size_t i = 0; i < count; i++)
         {
             const auto mode = internal::mapVkPresentMode(presentModes[i]);
```

Comment:
uhm this now broke, resize actually creates items inside the vector. and doing emplace back afterwards then doesn't overwrite those ofc

Reply:
oh oops, you're right